src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const fromThreeDaysDiscount = 20;
+  const fromSevenDaysDiscount = 50;
```

Comment:
```suggestion
  const smallDiscount = 20;
  const largeDiscount = 50;
```
Usually 2-3 words is enough to describe variable